azerothcore/azerothcore-wotlk
# fix(DB/Procs): Shaman's Elemental Focus should also proc off healing …

…spells (Healing Wave, Lesser Healing Wave, etc.)

"Elemental Focus (Elemental): Now also can proc off Lesser Healing Wave and Healing Wave, and can be used on all healing spells."
https://wowwiki-archive.fandom.com/wiki/Patch_3.0.2#Shaman



## Changes Proposed:
-  Allow Shaman's Elemental Focus to proc off healing wave/lesser healing wave, as it should according to patch notes
-  

## Issues Addressed:

- Closes 

## SOURCE:

Patch notes describing this functionality:
"Elemental Focus (Elemental): Now also can proc off Lesser Healing Wave and Healing Wave, and can be used on all healing spells."
https://wowwiki-archive.fandom.com/wiki/Patch_3.0.2#Shaman

Patch notes describing  revision or removal of this functionality during WotLK:
not found (looked through all pathces for 3.3.5a as found here: https://wowwiki-archive.fandom.com/wiki/Patches/3.x)
## Tests Performed:

- Tested in-game.
- 


## How to Test the Changes:


1. Create a shaman, learn the Elemental Focus talent from the elemental spec tree 
2. aura 54283 (30% crit)
3. cast healing wave, check that it also procs from healing spells now

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
